view/frontend/web/js/view/delivery-options.js

```diff
@@ -203,6 +211,40 @@ define(
         deliveryOptions.triggerEvent(deliveryOptions.updateDeliveryOptionsEvent);
       },
 
+      /**
+       * Checks the first level properties of two objects to see whether they are the same.
+       *
+       * @param {Object} object1
+       * @param {Object} object2
+       * @returns {boolean}
+       */
+      shallowObjectsAreTheSame: function(object1, object2) {
```

Comment:
Weet je zeker dat "shallow" goed genoeg is? er zitten nooit objecten in de properties van object1 of object2?

Reply:
Ik heb hier een kortere snippet die dit doet gevonden: https://dmitripavlutin.com/how-to-compare-objects-in-javascript/

Reply:
Ja beter iets bestaands gebruiken was ook mijn eerste gedachte maar ik schreef deze shallow functie sneller dan dat ik kon opzoeken of er al dergelijke functionaliteit in zat. Bedankt voor de link, nu is t beter.